packages/rocketchat-api/server/v1/users.js

```diff
@@ -271,6 +271,77 @@ RocketChat.API.v1.addRoute('users.createToken', { authRequired: true }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('users.getPreferences', { authRequired: true }, {
+	get() {
+		const user = this.userId;
```

Comment:
@jgtoriginal did you tested this? You will need to execute a find to get the user's object when you have the user's id

Reply:
@rodrigok no worries, will get to it at some point today and let you know.
Thanks very much!

Reply:
this has been solved by @MarcosSpessatto on https://github.com/RocketChat/Rocket.Chat/pull/9457/commits/19a8ad6ea3a889ca2c75271466b25ad273401d11